### PR TITLE
py.test panics about not finding the version

### PR DESCRIPTION
**Pull request by kaushik94**
Please take a look at #228 . This fixes it.

### CODE REVIEW

**Comment by wkerzendorf** [comment]
@embray is this sensible? if so should it go to package-template

@kaushik94 as an explanation TARDIS uses the package-template of astropy and if this is a good fix then it should also be fixed there.

**Comment by kaushik94** [comment]
@wkerzendorf  IMHO the problem might not be due to package-template, not quite sure why but when we don't supply the version number to astropy it tries to fetch it from the config file but in this case it isn't able to. Moreover the PR I submitted is merely a hack and not a solution, we should still dig into whats happening. The reason I came up with a quick fix is because `py.test` isn't working and its not possible to test code locally without travis. I think this could act as a temporary solution till we find the actual culprit.

**Comment by wkerzendorf** [comment]
run the tests using `python setup.py test`. That's how they're designed to be run. let me know if the problem still persists.

File: tardis/_astropy_init.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def test(package=None, test_path=None, args=None, plugins=None,
     if not os.environ.get('ASTROPY_SKIP_CONFIG_UPDATE', False):
         config_dir = os.path.dirname(__file__)
         try:
-            config.configuration.update_default_config(__package__, config_dir)
+            config.configuration.update_default_config(__package__, config_dir, version=__version__)
```
Comment (embray) [inline tardis/_astropy_init.py]: Weird. I don't think this should be necessary, but I haven't really looked closely at the problem yet.  You should be getting a `version.py` generated when you run `setup.py`.

**Comment by wkerzendorf** [comment]
@embray this is likely the problem. I think @kaushik94 is running `pytest` without `setup.py`

**Comment by kaushik94** [comment]
@wkerzendorf  I had run both `python setup.py install` and `python setup.py test` and I also got the version.py but still get the same error.
```
==================== 98 passed, 12 skipped in 5.36 seconds =====================
kaushik@ghost:~/Desktop/soc/tardis$ cd tardis
kaushik@ghost:~/Desktop/soc/tardis/tardis$ cd tests/
kaushik@ghost:~/Desktop/soc/tardis/tardis/tests$ py.test
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/_pytest/config.py", line 543, in importconftest
    mod = conftestpath.pyimport()
  File "/usr/local/lib/python2.7/dist-packages/py/_path/local.py", line 629, in pyimport
    __import__(pkgpath.basename)
  File "/home/kaushik/Desktop/soc/tardis/tardis/__init__.py", line 7, in <module>
    from ._astropy_init import *
  File "/home/kaushik/Desktop/soc/tardis/tardis/_astropy_init.py", line 125, in <module>
    config.configuration.update_default_config(__package__, config_dir)
  File "/usr/local/lib/python2.7/dist-packages/astropy/config/configuration.py", line 844, in update_default_config
    version = mod.__version__
AttributeError: 'module' object has no attribute '__version__'
ERROR: could not load /home/kaushik/Desktop/soc/tardis/tardis/conftest.py
```

**Comment by wkerzendorf** [comment]
you do not need to run the install. I don't know how the astropy version is. I suggest using anaconda as described in the manual.

**Comment by wkerzendorf** [comment]
@kaushik94 i fixed it I think

**Comment by wkerzendorf** [comment]
@kaushik94 if it's fixed - close this (but still put this PR in your application :wink: )

**Comment by kaushik94** [comment]
@wkrzendorf are you talking about PR #250 ? That looks like a cleaner fix than mine, will pull your changes and close this now. Will let you know if I run into trouble.
>(but still put this PR in your application :wink: )

Haha sure :laughing:

**Comment by kaushik94** [comment]
also take a look at #234 when you find some time

**Pull request closed without merging**